.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
If I'm understanding this correctly, I think maybe we shouldn't commit VSCode config to the repo. That seems like it would be specific to each person